Common/Sources/Extensions/HyperwalletTransferMethod.swift

```diff
@@ -41,12 +41,26 @@ extension HyperwalletTransferMethod: GenericCellConfiguration {
             return String(format: "%@%@",
                           "to".localized(),
                           getField(TransferMethodField.email.rawValue) ?? "")
-
+        case "PREPAID_CARD":
+            let cardBrand = getField(TransferMethodField.cardBrand.rawValue)?.lowercased().localized() ?? ""
+            let cardNumber = getField(TransferMethodField.cardNumber.rawValue)?.suffix(startAt: 4) ?? ""
+            let dots = Array(repeating: "\u{2022}", count: 4).joined()
+            return String(format: "%@ %@ %@", cardBrand, dots, cardNumber)
+            
         default:
             return String(format: "%@%@",
                           "endingIn".localized(),
                           getField(TransferMethodField.bankAccountId.rawValue)?
                             .suffix(startAt: 4) ?? "")
         }
     }
+
+    /// Checking for transfer method is prepaid card or not
+    /// - Returns: True if transfer method is prepaid card
+    public func isItPrepaidCard() -> Bool {
+        if type == "PREPAID_CARD" {
```

Comment:
instead of hardcoding "PREPAID_CARD" use `TransferMethodType.prepaidCard.rawValue`.
Update the entire func to 
`func isPrepaidCard() -> Bool {
        return type == TransferMethodType.prepaidCard.rawValue
    }`

Reply:
Updated the changes